nights/2015-11-01-README.md

```diff
@@ -1,32 +1,24 @@
 # 2015-11-01
 
-## Reviewed by:   your_name_here
+## Reviewed by:   Isobel Snellenberger
```

Comment:
Please add a link to the "image gallery" for this night (see the issue for this night for the link).